openmp/libomptarget/src/device.cpp

```diff
@@ -66,7 +66,7 @@ int HostDataToTargetTy::addEventIfNecessary(DeviceTy &Device,
 
 DeviceTy::DeviceTy(PluginAdaptorTy *RTL, int32_t DeviceID, int32_t RTLDeviceID)
     : DeviceID(DeviceID), RTL(RTL), RTLDeviceID(RTLDeviceID),
-      PendingCtorsDtors(), PendingGlobalsMtx(), MappingInfo(*this) {}
+      MappingInfo(*this) {}
```

Comment:
i think this is extraneous ? duplicated ?

Reply:
It's just deleting the previous globals, don't think so.

Reply:
oops, was downstream merge foolishness, you are correct , its fine